helmfile/charts/gatekeeper/podsecuritypolicies/values.yaml

```
@@ -0,0 +1,47 @@
+# Add pods that should override the default constraints
+# <namespace>:
+#   <service-name>:
+#     labelsSelector:
+#       foo: bar
+#     # Every "allow" is an override, and is optional.
+#     # If a override is NOT part of "allow", the default constraint is applied
+#     allow:
+#       allowedCapabilities:
+#         - NET_BIND_SERVICE
+#         - ...
+#       # Also allows for ports 1-65535, limit with network policies
+#       hostNetwork: <true/false>
+#       hostNamespace: true
+#       privilegedEscalation: true
+#       privileged: true
+#       allowedHostPaths:
+#         - pathPrefix: /proc
+#           readOnly: true
+#         - ...
+#       volumes:
+#         - emptyDir
+#         - ...
+#       runAsUser:
+#         runAsUser:
+#           rule: MustRunAsNonRoot
+#         runAsGroup:
+#           rule: MustRunAs
+#           ranges:
+#             - max: 65535
+#               min: 1
+#         supplementalGroups:
+#           rule: MustRunAs
+#           ranges:
+#             - min: 1
+#               max: 65535
+#         fsGroup:
+#           rule: MustRunAs
+#           ranges:
+#             - min: 1
+#               max: 65535
+#       sysctl:
+#         - ...
+constraints: {}
+
+# List of pod labels that mutation should be exempted from
+mutation: []
```

Could you make the name a bit more obvious that this excludes things from the mutation? Something like this:
```suggestion
mutationExceptions: []
```

---

Can do :+1: